controllers/apiController.js

```diff
@@ -61,7 +61,7 @@ exports.fleetAtAGlance = function(req, res) {
 
 //Store a new banner
 exports.addBanner = function(req, res){
-    if(req.isAuthenticated() && req.user.roleNumeric > 0){
+    if(req.isAuthenticated() && req.user.role.numeric > 0){
```

Comment:
How is the data transitioning working for this? Existing data that has to transfer from `roleNumeric` to `role.numeric`?

Reply:
Can we make this an automatic transition? Like can we add a helper function called `isAuthenticated()` or whatever the situation calls for here?

Reply:
Two options:
1. Drop users table and force everyone to re-register. This will only really affect FCs. 
2. Make and run  a script to change the user objects. Note that changes to the model include:
 - roles and role numerics are now an object
 - Alliance and corporations are now objects with a name and ID only.
 - Notes is now an empty array not a string
 - Removed the ships array 
 - Related characters has been removed
 - Token and Date have been moved to the end